packages/tools/client-debug-view/src/components/AudienceView.tsx

```diff
@@ -184,10 +209,15 @@ function HistoryView(props: HistoryViewProps): React.ReactElement {
 			styles={{
 				root: {
 					overflowY: "auto",
+                    height: "300px"
```

Comment:
Do we need to provide a height here? Should we maybe make it a `minHeight` instead so it still fills the available vertical space when it can?

Reply:
<img width="296" alt="image" src="https://user-images.githubusercontent.com/111468570/206592924-30c91150-0d33-4271-8b86-f1ffc86306a9.png">

<img width="329" alt="image" src="https://user-images.githubusercontent.com/111468570/206592967-ef4031f2-d416-4896-9959-d8f7110aa80a.png">


If apply only the `minHeight` to `<Stack>`'s `styles`, `historyViews` is once again not available through scrolling the elements. 

https://stackoverflow.com/questions/21227287/make-div-scrollable
https://stackoverflow.com/questions/9707397/making-a-div-vertically-scrollable-using-css

Answers tend to differ on internet, (which is totally understandable as CSS has gazillion ways to do one thing), but to me, setting `overflowY: "auto" and `maxHeight` seemed to be the general consensus.

Reply:
Ah, okay. I guess that makes sense. The current behavior seems fine to me then (and eventually we'll have a designer tell us to redo all of this anyways 😋) .

Reply:
Thanks for the feedback! But I will make sure to look into it and make edits if I find a better solution 👍